src/_guides/language/language-tour.md

```diff
@@ -2630,7 +2630,7 @@ class Point {
   final num y;
   final num distanceFromOrigin;
 
-  Point(x, y)
+  Point(num x, num y)
```

Comment:
This looked really weird at first, but now I get it. (You aren't referring to the instance variables here; it looks weird because this isn't idiomatic Dart.)